src/main/java/configuration/Config.java

```diff
@@ -53,7 +53,7 @@ public static void init() throws Exception {
         KAFKA_BROKER = getString(dotenv, "KAFKA_BROKER");
         GROUP_ID = getString(dotenv, "GROUP_ID");
         TARGET_BASE_URL = getString(dotenv, "TARGET_BASE_URL");
-        TOPICS_ROUTES = getStringMap(dotenv, "TOPICS_ROUTES");
+        TOPICS_ROUTES = new TopicsRoutes(getStringMap(dotenv, "TOPICS_ROUTES"));
```

Comment:
This shouldn't come from config but from class TopicRoutes
The class should be injected in HttpTarget constructor, and initialized in Main with a Map topicRoutes